kai/config.toml

```diff
@@ -21,8 +21,13 @@ password = "dog8code"
 #   OpenAI GPT 4
 #     provider = "OpenAI"
 #     args = { model_id = "gpt-3.5-turbo" }
-provider = "IBMGranite"
-args = { model_id = "ibm/granite-13b-chat-v2" }
+# provider = "IBMGranite"
+# args = { model_id = "ibm/granite-13b-chat-v2" }
```

Comment:
Is the intent to not have a model configured by default?

Reply:
Mistype, fixing